### ISSUE TITLE
tap method blocks the main thread

### ISSUE THREAD

**Issue by k1r0s** [Einabit/client.java#9]
It should create two subscriptions, it rather creates one subscription bcoz the thread is blocked:

shouldn't start a new thread here? https://github.com/Einabit/client.java/blob/main/src/main/java/com/einabit/client/EinabitClient.java#L87-L90

sample code:
```java
import com.einabit.client.EinabitClient;
import com.einabit.client.EinabitServerListener;

public class Main {

  private static final String[] VARIABLES = {"temp1", "crg1"};
  private static final String ENV_HOST = "HOST";

  public static void main(String[] args) {

    final EinabitClient client = EinabitClient.builder().host(System.getenv(ENV_HOST)).build();

    for (String auxvarname: VARIABLES) {

      client.tap(auxvarname, new EinabitServerListener () {
        @Override
        public void onSubscribe(String auxval) {
          System.out.println(auxvarname + " updated, value: " + auxval);
        }
      });
      System.out.println("Successfully subscribed to " + auxvarname);
    }
  }
}

```

expected output
```
demo-java-postgre-sandbox-1      | Server started
demo-java-postgre-sandbox-1      | New connection started
demo-java-postgre-sandbox-1      | Received command tap,temp1
demo-java-postgre-sandbox-1      | New connection started
demo-java-postgre-sandbox-1      | Received command tap,crg1
demo-java-postgre-application-1  | Successfully subscribed to temp1
demo-java-postgre-application-1  | Successfully subscribed to crg1
demo-java-postgre-application-1  | temp1 updated, value: 1697025787941,2
demo-java-postgre-application-1  | temp1 updated, value: 1697025788141,1
demo-java-postgre-application-1  | crg1 updated, value: 1697025788141,3
demo-java-postgre-application-1  | crg1 updated, value: 1697025788141,3
demo-java-postgre-application-1  | temp1 updated, value: 1697025788141,1

```
received output
```
demo-java-postgre-sandbox-1      | Server started
demo-java-postgre-sandbox-1      | New connection started
demo-java-postgre-sandbox-1      | Received command tap,temp1
demo-java-postgre-application-1  | temp1 updated, value: 1697025787941,2
demo-java-postgre-application-1  | temp1 updated, value: 1697025788141,1
demo-java-postgre-application-1  | temp1 updated, value: 1697025788542,2
demo-java-postgre-application-1  | temp1 updated, value: 1697025788942,1
demo-java-postgre-application-1  | temp1 updated, value: 1697025789543,2
demo-java-postgre-application-1  | temp1 updated, value: 1697025789945,1
demo-java-postgre-application-1  | temp1 updated, value: 1697025791548,2
demo-java-postgre-application-1  | temp1 updated, value: 1697025791749,3
... it never releases main thread
```

**Comment by k1r0s** [comment]
if useful

docker-compose.yml

```yml
version: "3.9"
services:
  application:
    build:
      context: app/
      dockerfile: Dockerfile.app
    environment:
      HOST: sandbox
    depends_on:
      - sandbox
  sandbox:
    image: einabit/sandbox
    environment:
      CONFIG: |
        crg1:
          value:
            min: 20
            max: 40
            step: 2
          interval:
            ms: 1200
            skip: .5
        temp1:
          value:
            min: 1
            max: 50
            step: 1
          interval:
            ms: 200
            skip: .3
```

**Comment by nicomness** [comment]
You can find a working example of how to achieve running tap command in a separate thread without blocking the app here https://github.com/Einabit/client.java/pull/11/files

**Comment by k1r0s** [comment]
closed by #5
